server/plugin/webhook.go

```diff
@@ -1382,3 +1392,90 @@ func (p *Plugin) postReleaseEvent(event *github.ReleaseEvent) {
 		}
 	}
 }
+
+func (p *Plugin) postDiscussionEvent(event *github.DiscussionEvent) {
+	repo := event.GetRepo()
+
+	subs := p.GetSubscribedChannelsForRepository(repo)
+	if len(subs) == 0 {
+		return
+	}
+
+	newDiscussionMessage, err := renderTemplate("newDiscussion", event.GetDiscussion())
+	if err != nil {
+		p.client.Log.Warn("Failed to render template", "error", err.Error())
+		return
+	}
+
+	for _, sub := range subs {
+		if !sub.Discussions() {
+			continue
+		}
+
+		if p.excludeConfigOrgMember(event.GetSender(), sub) {
+			continue
+		}
+
+		post := &model.Post{
+			UserId:  p.BotUserID,
+			Type:    "custom_git_discussion",
+			Message: newDiscussionMessage,
+		}
```

Comment:
you can use the `makeBotPost` function here, as you used in the func below

Reply:
updated in commit 3a637da